core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -145,8 +145,16 @@ public static Snapshot oldestAncestorOf(long snapshotId, Function<Long, Snapshot
   }
 
   public static Iterable<Snapshot> ancestorsOf(long snapshotId, Function<Long, Snapshot> lookup) {
+    return ancestorsOf(snapshotId, lookup, true);
+  }
+
+  public static Iterable<Snapshot> ancestorsOf(
```

Comment:
this method only exists so that we have consistent error reporting in https://github.com/apache/iceberg/blob/1521296dd253c829dedc3e83ffda4cd2622667e2/core/src/test/java/org/apache/iceberg/TestSnapshotLoading.java#L169 for V1 and V2 tables (because we want to skip the validation of the starting snapshot id).